module/VuFind/src/VuFind/Search/Base/Params.php

```diff
@@ -1740,14 +1740,8 @@ public function hasHiddenFilter($filter)
     {
         // Extract field and value from URL string:
         [$field, $value] = $this->parseFilter($filter);
-
-        if (
-            isset($this->hiddenFilters[$field])
-            && in_array($value, $this->hiddenFilters[$field])
-        ) {
-            return true;
-        }
-        return false;
+        return isset($this->hiddenFilters[$field])
+        && in_array($value, $this->hiddenFilters[$field]);
```

Comment:
I would further simplify this to:

```suggestion
        return in_array($value, $this->hiddenFilters[$field] ?? []);
```